wezterm-gui/src/commands.rs

```diff
@@ -681,20 +681,28 @@ pub fn derive_command_from_key_assignment(action: &KeyAssignment) -> Option<Comm
             menubar: &["View"],
             icon: Some("md_fullscreen"),
         },
-        ToggleAlwaysOnTop => CommandDef {
+        SetWindowLevel(SetWindowLevelLevel::AlwaysOnTop) => CommandDef {
             brief: "Always on Top".into(),
             doc: "Toggles the window between floating and non-floating states to stay on top of other windows.".into(),
             keys: vec![],
             args: &[ArgType::ActiveWindow],
-            menubar: &["Window"],
+            menubar: &["Window", "Level"],
+            icon: None,
+        },
+        SetWindowLevel(SetWindowLevelLevel::Normal) => CommandDef {
```

Comment:
![CleanShot 2023-11-30 at 03 08 16@2x](https://github.com/wez/wezterm/assets/16429579/d4109596-6bb9-4fd3-b62b-7dc721ac6d73)

Since now there are 2 options (3 with the default level) I tought it was good to have a submenu with selectable levels instead of 2 toggles. Also it would be nice (in both cases) to be able to display a tick when selected something like the last item of window menu 

![CleanShot 2023-11-30 at 03 07 13@2x](https://github.com/wez/wezterm/assets/16429579/d9e4307f-c53e-46c4-8ebc-e78c74324c4a)

Reply:
Also naming i believe it can be improved

Reply:
actually after some reasoning maybe the 2 options "Toggle Always on Top"  and "Toggle Always on Bottom" have a better UX (thinking of assigning a shortcut, having a toggle is easier than selecting a level). btw having the tick on the left of the option would improve the ux a lot since you can understand the current state of the window.

Reply:
Having ticks is out of scope of this PR. I'm not sure how feasible it is to accommodate that concept, because a number of the states potentially require some kind of async call to resolve, and it may not be possible to handle those in all contexts, or even to invalidate and update in all contexts.  So, I like the idea, but let's put it out of mind for the moment!

Reply:
re: Toggle, I think that having both a `Set` and a `Toggle` operation will be important.  Some users will want to be sure that they can set a specific state regardless of the current state in some actions, while others will want to assign a key press to the toggle behavior.